### PR TITLE
Updating exsisting Lumia 520 device details and adding a Lumia 950.

### PR DESCRIPTION
**Pull request by andysterland**
Three changes:
1. Updating the UA string for Edge to reflect the latest Edge public release (Edge 13).
2. Updating the existing Nokia Lumia 520's device-pixel-ratio to 1.5
3. Adding a for the new Microsoft 950. Which is not shown by default.  

One trivial style change:
1. Moving the IE8 UA to be after the IE9 UA so the IE UA's are sorted descending.

### CODE REVIEW

**Comment by andysterland** [comment]
I signed it!

**Comment by dgozman** [comment]
Thank you!

File: devices.json [hunk 1]
```diff
@@ -528,6 +528,42 @@
...
+                "vertical": {
+                    "width": 360,
+                    "height": 640
+                }
+            },
+            "capabilities": [
+                "touch",
+                "mobile"
+            ],
+            "user-agent": "Mozilla/5.0 (Windows Phone 10.0; Android 4.2.1; Microsoft; Lumia 950) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/46.0.2486.0 Mobile Safari/537.36 Edge/13.10587",
```
Comment (jacobrossi) [inline devices.json:547]: The build that ships on the Lumia 950 was 13.10586  not 10587

Comment (paulirish) [inline devices.json:547]: ill fix that in master.

Comment (paulirish) [inline devices.json:547]: ah, andy fixed it in a subsequent commit, bumping to 14.x

Comment (andysterland) [inline devices.json:547]: Yea there was a couple of other issues with the UA but I believe that the latest update is the "right one" (tm) and I've checked it with folk.

**Pull request merged**